### PR TITLE
Rrose/dice game farkle/

### PR DESCRIPTION
**Pull request by rmrose78**

### CODE REVIEW

**Comment by rmrose78** [comment]
I should have added the pull request template through GitHub and not on VSCode. PR template should show in the PRs following this one. Added scoring to the game. current round points, as well as point update on dice select is working. points are allocated to the active player when the hold button is pressed.

**Comment by rmrose78** [comment]
I also added the file that activates Prettier on the project.

**Comment by axecopfire** [comment]
You can click on the description of the PR and edit it to add the template

File: .github/PULL REQUEST TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Description
```
Comment (axecopfire) [inline .github/PULL REQUEST TEMPLATE.md:1]: Idk if this file is named properly

File: index.html [hunk 2]
```diff
@@ -15,9 +15,9 @@
     <div class="main">
       <div class="Reset"></div>
       <h1>Farkle</h1>
-      <div class="currentPoints-container">
+      <div class="points-current-container">
         <h3>Points:</h3>
-        <h3 id="currentPoints">0</h3>
+        <h3 id="points-current">0</h3>
```
Comment (axecopfire) [inline index.html:20]: I appreciate the naming consistency

File: main.js [hunk 3]
```diff
@@ -1,100 +1,233 @@
...
-    activePlayer_1.classList.toggle("active")
-    activePlayer_2.classList.toggle("active")
-
-    //reset on roundTotal and selectedTotal points
-    points.roundTotal = 0
-    points.selectedTotal = 0
-
-})
+document.getElementById("hold-button").addEventListener("click", () => {
+  if (activePlayer1.classList[1] === "active") {
```
Comment (axecopfire) [inline main.js:101]: This works for now. But it's a bit fragile. Maybe [`Array.includes`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) would be better?

File: main.js [hunk 4]
```diff
@@ -1,100 +1,233 @@
...
+        diceSelected.push(dice.positionValues[die]);
+      }
+    }
+    return diceSelected;
+  },
 
+  freq: function (diceSelected) {
+    let freqObj = {};
+
+    diceSelected.forEach((die) => {
```
Comment (axecopfire) [inline main.js:153]: We'll go over this next week but it looks like ur turning an array into an object. I personally would use [`Array.reduce`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce). Tho I understand `.reduce` is the most complicated of the Higher Order Array Functions, so it'd make sense if you don't end up choosing it.

File: main.js [hunk 5]
```diff
@@ -1,100 +1,233 @@
...
+    });
+    return freqObj;
+  },
+
+  scoring: function (freqObj) {
+    freqKeys = Object.keys(freqObj);
+
+    switch (true) {
+      // 6 of a kind
+      case freqObj[`${freqKeys[0]}`] === 6:
```
Comment (axecopfire) [inline main.js:165]: I'd like to see `${freqKeys[0]}` assigned to a variable. Idk what the value is supposed to be. Also, it doesn't have to be in a template literal `freqObj[freqKeys[0]]` works

Comment (axecopfire) [reply main.js:165]: Also, why is this not in the loop? What's special about the `0`th case?

**Review by axecopfire — APPROVED** [review]

**Pull request merged**